exosphere-shared/src/docker-helper.ls

```diff
@@ -48,6 +48,38 @@ class DockerHelper
       done null, map((.RepoTags |> head |> (.split ':') |> head), images)
 
 
+  @get-dangling-images = (done) ->
+    docker.list-images {"filters": '{"dangling": ["true"]}'}, (err, images) ->
+      | err => done err
+      done null, images
+
+
+  @force-remove-images = (images, done) ->
+    if images.length == 0 then done! else
+      num-removed = 0
+      for image in images
+        docker.get-image(image.Id).remove {force:true}, (err) ->
+          | err => done err
+          num-removed += 1
+          if num-removed == images.length then done!
```

Comment:
For running multiple async processes in series, we use: https://caolan.github.io/async/docs.html#mapSeries. Take a look at how we do this on the `exo-setup` command: https://github.com/Originate/exosphere/blob/master/exo-setup/src/app-setup.ls#L46.